### PR TITLE
Revert #1455

### PR DESCRIPTION
**Pull request by simon33-2**
## Overview
Reverts PR #1455. 

## Functional Changes
Allows users to target the same territory with two different rockets.

## Manual Testing Performed
- ..None

## Additional Review Notes

My local build isn't working

### CODE REVIEW

**Comment by simon33-2** [comment]
I've forgotten how to do this all. I think this will work but I haven't tested it. I want to download the built version and test that. How do I do that again?

This also allows you to see the result of the first rocket fired before targetting a second one. This leaves a bad taste to implement a stupid rule. However, the way it presently is is also poor.

**Comment by RoiEXLab** [comment]
@simon33-2 The easiest way is probably to use an IDE based launch.
You could of course also just use `./gradlew :game-core:run` (not sure if `./gradlew run` still works) to test everything via gradle.
Now that I'm thinking about it: You may need to install lombok for your IDE in order for some stuff to compile correctly.

**Comment by ssoloff** [comment]
Yeah, `./gradlew :game-core:run` would be easiest.  @simon33-2, if that doesn't work for some reason, I can fork your branch and create the release artifacts in my repo for you to test.

File: game-core/src/main/java/games/strategy/triplea/delegate/RocketsFireHelper.java [hunk 1]
```diff
@@ -103,14 +100,12 @@ private static void fireWW2V2(
       // Ask the user where each rocket launcher should target.
```
Comment (ssoloff) [inline game-core/src/main/java/games/strategy/triplea/delegate/RocketsFireHelper.java]: This comment was added in #1455.  Is it still applicable after this revert?

Comment (simon33-2) [reply game-core/src/main/java/games/strategy/triplea/delegate/RocketsFireHelper.java]: Arguably it is. The user is asked where the rocket should target still. It just doesn't store the target - it fires it immediately. I'd be ok with it being left or being removed.

**Pull request merged**